wiki/osu!stream/en.md

```diff
@@ -22,48 +22,47 @@ In 2023, twelve years after its initial release, osu!stream was finally [ported
-### iOS App Store method (suggested)
+### App Store method (suggested)
 
-**Make sure the iOS device can connect to the Internet (either by Wi-Fi or 3G/4G).**
+*Note: Make sure the device can connect to the Internet (either by Wi-Fi or 3G/4G).*
 
-- Go to "App Store" from the iOS device
+- Go to "App Store" from the iOS device or "Play Store" on Android.
 - Search for osu!stream
 - Press and install osu!stream
-  - If prompted to key-in the Apple ID and the password, please do so.
-- osu!stream can be played after the download and auto-installation is completed.
-  - It is suggested to read the [Adding Beatmaps](#adding-beatmaps) section too to get the latest free songs pack. The current download contains only 2 songs.
+  - If prompted to type in any account details, please do so.
+- osu!stream can be played after the download and installation is completed.
+- It is recommended to read the [Adding Beatmaps](#adding-beatmaps) section too to get the latest free songs pack. The current download contains only 2 songs.
 
-### USB method
+### USB method (iOS-only)
 
-**Device must have [iTunes](https://www.apple.com/itunes/) installed.**
+*Notice: For this method, [iTunes](https://www.apple.com/itunes/) must be installed.*
 
-- Go to [this link and download it](http://itunes.apple.com/us/app/osu!stream/id436952197?ls=1&mt=8)
-  - If requested for Apple ID, key-in the Apple ID and the password.
-- Plug-in the iOS device to the device (that has osu!stream).
+- Go to [this link and download it](http://itunes.apple.com/us/app/osu!stream/id436952197?ls=1&mt=8).
+  - If requested for Apple ID, type in the Apple ID and the password.
+- Plug-in the iOS device to the device on which osu!stream is installed.
 - Transfer the osu!stream files using iTunes.
 - Unplug the iOS device and osu!stream can be played.
 
 ## Adding Beatmaps
 
-**Make sure the iOS device can connect to the Internet (either by Wi-Fi or 3G/4G).**
+*Note: Make sure the device can connect to the Internet (either by Wi-Fi or 3G/4G).*
 
 - Open osu!stream.
-- Press the osu! symbol and it should be directed to the main menu (or tutorial if first time)
+- Press the osu! symbol to go to the main menu (or tutorial if playing for the first time)
 - Press the "Store" bar
   - Alternatively, head to "Play" and press "Download more songs!"
 - Select a desired pack.
 
 ## What is the difference?
 
-- **Playable on any iOS devices only.** (for now)
 - Beatmaps are made from scratch.
 - No account needed to download the song packs.
-- Can integrate osu!stream with a Twitter account.
-- High score may be sent anonymously to the site under the name of "Guest" (requires Internet connection).
+- osu!stream can be integrated with a Twitter account on iOS.
+- High scores may be submitted anonymously under the name of "Guest" (requires an internet connection).
 - The circle timer is replaced and moved to the bottom with a small border line which is coloured depending on the accuracy of the play. \[Bottom(Long)\]
```

Comment:
What does "Bottom(Long)" mean? Is this needed?

Reply:
yeah i'd say just delete that lol

Reply:
```suggestion
- The circle timer is replaced and moved to the bottom with a small border line which is coloured depending on the accuracy of the play.
```